tools/ui-components/src/index.ts

```diff
@@ -1,6 +1,6 @@
 // Use this file as the entry point for component export
-// export { Button } from './button';
 export { Alert, type AlertProps } from './alert';
+export { Button } from './button';
```

Comment:
```suggestion
// export { Button } from './button';
```
This should not matter, but we ended up with some unpredictable outcomes. I would prefer adding this during the component swap.

Reply:
Also, the related tests are not passing locally either. So they need to be updated accordingly.